fuel-tx/src/transaction/types/utxo_id.rs

```diff
@@ -28,6 +28,11 @@ pub struct UtxoId {
     output_index: u16,
 }
 
+#[cfg(feature = "da-compression")]
+impl fuel_compression::Compressible for UtxoId {
+    type Compressed = (crate::TxPointer, u16);
```

Comment:
Maybe it is better to define `CompressedUtxoId` instead of tuple

Reply:
Definitely. This is a draft PR for a reason.

Reply:
Done in b60e38a4